.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -4,7 +4,17 @@ on:
     branches:
       - main
       - prerelease_test
+    paths-ignore:
+      - '**.md'
+      - LICENSE
```

Comment:
We could add `LICENSE-TIMESCALE` as well to the list.

Reply:
Added `LICENSE*`.